doc/API.md

```diff
@@ -68,3 +88,41 @@ filtered by an optional `status` parameter.
 | Field         | Type                                       | Description               |
 | ------------- | ------------------------------------------ | ------------------------- |
 | `vaults`      | array of [vault resource](#vault-resource) | Vaults filtered by status |
+
+### `sigvault`
+
+The `sigvault` RPC Command executes the signing process of the Revault
+pre-signed transactions.
+
+#### Request
+
+| Parameter        | Type   | Description                         |
+| ---------------- | ------ | ----------------------------------- |
+| `vault_id`       | string | Unique id of the vault to sign      |
+| `only_emergency` | bool   | Sign only the emergency transaction |
+
+#### Response
+
+TODO: specify response
+
+### `spendvault`
+
+The `spendvault` RPC command creates a new spending request and creates
+a [PSBT](https://github.com/bitcoin/bitcoin/blob/master/doc/psbt.md) 
+spending the vault with the given ID and send it to cosigning
```

Comment:
They need to be able to spend multiple vaults at once, this should take an array

Reply:
This then should give also for each vault the specific parameters: like
```json
vaults: [{vault_id: "...", "<parameters>": {...}}]
```

Reply:
Yeah absolutely.

Reply:
maybe the parameters can be only an unsigned psbt